### PR TITLE
updating Azure functions to v3 for .net core 3.1 project

### PR DESCRIPTION
**Pull request by NinadKavimandan**

### CODE REVIEW

File: dotnet/aspnetcore31/functionApp/Application/SampleFunctionApp/SampleFunctionApp.csproj [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <TargetFramework>netcoreapp3.1</TargetFramework>
-    <AzureFunctionsVersion>v2</AzureFunctionsVersion>
+    <AzureFunctionsVersion>v3</AzureFunctionsVersion>
```
Comment (bishal-pdMSFT) [inline dotnet/aspnetcore31/functionApp/Application/SampleFunctionApp/SampleFunctionApp.csproj:4]: is this forced by .net core?

**Comment by bishal-pdMSFT** [comment]
do we need any change in the BD/RD?

File: dotnet/aspnetcore31/functionApp/Application/SampleFunctionApp/SampleFunctionApp.csproj (hunk 1 above)
Comment (NinadKavimandan) [reply dotnet/aspnetcore31/functionApp/Application/SampleFunctionApp/SampleFunctionApp.csproj:4]: No, .NET Core 3.1 wasn't supported in Azure functions V2, thus upgrading to V3. https://docs.microsoft.com/en-us/azure/azure-functions/functions-versions#languages

**Comment by NinadKavimandan** [comment]
No, changing this is enough. BD/RD are not failing, failure was once we call that function after getting deployed.

**Pull request merged**